.eslintrc.json

```diff
@@ -327,7 +327,7 @@
       }
     },
     {
-      "files": ["*.e2e.ts"],
+      "files": ["*.e2e.ts", "*.ts"],
```

Comment:
Maybe we can go for something like `e2e/**/*.ts` to make sure those rules are targeting `ts` files only inside `e2e` directory

Reply:
I tried many different ways to do this but none works. Only the "*.ts" works. 

I dont think that it can create any issues, because this only targets playwright-like methods etc, so non-playwright ts files should be safe. Let me know what you think